live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -266,3 +266,45 @@
       ## Record command exit failures (execve result is not command result)
       -a exit,always -F a0!=0 -S exit_group
       -a exit,always -F a0!=0 -S exit
+
+#
+# By default, the ulimit for core files is set to 0, and the default
+# filename and location for a core file is 'core' in the cwd. Update
+# limits.conf to allow processes running as root or a regular user to
+# make core files.
+#
+- lineinfile:
+    create: yes
+    dest: /etc/security/limits.conf
+    line: "{{ item.domain }} soft core unlimited"
+  with_items:
+    - { domain: "root" }
+    - { domain: "*" }
+
+#
+# Create a world writeable directory for application core dumps.
+#
+- file:
+    path: /var/crash
+    state: directory
+    mode: 0777
+
+#
+# Update sysctl.conf such that application cores are written into
+# /var/crash and tagged with the execname, process id, and seconds since
+# the epoch.
+#
+# Additionally in this file, we force the kernel to assume memory
+# allocations will succeed until we actually run out of memory. The
+# default heuristic can cause failure to generate an hprof dump on OOM
+# in the stack. Allowing overcommit lets the fork(2) succeed despite not
+# having enough memory which allows the script that generates the hprof
+# dump to run.
+#
+- lineinfile:
```

Comment:
you could add a `regexp` field, so that any default value will get replaced by the new value. I'm OK leaving it out, but that'd make things a little more elegant; e.g. something like this:
```
+- lineinfile:
+    create: yes
+    dest: /etc/sysctl.conf
+    regexp: "^#?{{ item.key }} ="
+    line: "{{ item.key }} = {{ item.value }}"
+  with_items:
+    - { key: 'kernel.core_pattern', value: '/var/crash/core.%e.%p.%t' }
+    - { key: 'vm.overcommit_memory', value: '1' }
```